docs/topics/proc_installing-launcher-tool-addon.adoc

```diff
@@ -15,6 +15,14 @@ link:https://docs.openshift.org/latest/minishift/using/addons.html[Add-ons] enab
 
 .Procedure
 
+. Ensure you are running a minishift instance which has at least 4 GB memory to run launcher
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ minishift config set memory 8192
```

Comment:
FYI, we edited this in https://github.com/fabric8-launcher/launcher-documentation/pull/1177 to 4096, because saying "at least 4 GB" and then showing 8 GB is just confusing.

Reply:
And 5.1 uses 8192 for installing as an add-on...

Reply:
What's "5.1"?

Reply:
Sorry! I guess the section numbers don't appear on the upstream/donwstreem renditions. You can see it here: https://access.redhat.com/documentation/en-us/red_hat_openshift_application_runtimes/1/html/install_and_configure_the_fabric8_launcher_tool/installing-launcher-tool_minishift in step 1.

Reply:
Ah I see! Sorry, it didn't even occur to me that you might be referring to chapter number :-)

Yea the current prod rendering shows 8192, but it shows 4096 on stage.